backend/internal/userschema/store/queries.go

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright (c) 2025, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+// Package store defines the database queries for user schema management operations.
+package store
+
+import "github.com/asgardeo/thunder/internal/system/database/model"
+
+var (
+	// QueryGetUserSchemaCount retrieves the total count of user schemas.
+	QueryGetUserSchemaCount = model.DBQuery{
+		ID: "ASQ-USER_SCHEMA-001",
+		PostgresQuery: `
+			SELECT COUNT(*) AS total 
+			FROM user_schemas`,
+		SQLiteQuery: `
+			SELECT COUNT(*) AS total 
+			FROM user_schemas`,
+	}
+
+	// QueryGetUserSchemaList retrieves a paginated list of user schemas.
+	QueryGetUserSchemaList = model.DBQuery{
```

Comment:
Do we need to define two queries just for the parameters? IIRC sqlite supports both `$` and `?`.

https://www.sqlite.org/lang_expr.html#:~:text=the%20token%20%22NULL%22.-,4.%20Parameters,-A%20%22variable%22%20or

Reply:
Fixed with https://github.com/asgardeo/thunder/compare/2359bb281523048120acb1f8c4d2b4ce08b10f5f..d0b7099012c7c17094c1feb4e8baa826f84603be